### PR TITLE
ADD: Tests for Numpy input for image and added type conversion to uint8

### PR DESCRIPTION
**Pull request by sihrc**
PIL supported numpy array inputs

### CODE REVIEW

File: indicoio/utils/image.py [hunk 1]
```diff
@@ -33,20 +33,24 @@ def image_preprocess(image, size=(48,48), batch=False):
...
         )
-        outImage = process_list_image(image)
+        out_image = process_list_image(image)
     elif isinstance(image, Image.Image):
-        outImage = image
+        out_image = image
     elif type(image).__name__ == "ndarray": # image is from numpy/scipy
-        out_image = Image.fromarray(image)
+        try:
+            out_image = Image.fromarray(image.astype("uint8"))
```
Comment (madisonmay) [inline indicoio/utils/image.py]: Could you fill me in on how this command behaves when the image is of type float (and 0-1 bounded?).

**Comment by madisonmay** [comment]
On a very related note, https://uploads.intercomcdn.com/i/o/1405448/8ba13e8ac605319bb87c7de9/Capture.PNG.  Might make sense to push a fix for this issue in the same PR.

**Comment by sihrc** [comment]
@madisonmay got it. latest commit

**Comment by sihrc** [comment]
@madisonmay we probably want to push these fixes soon

File: indicoio/utils/image.py [hunk 2]
```diff
@@ -33,20 +33,26 @@ def image_preprocess(image, size=(48,48), batch=False):
...
         )
-        outImage = process_list_image(image)
+        out_image = process_list_image(image)
     elif isinstance(image, Image.Image):
-        outImage = image
+        out_image = image
     elif type(image).__name__ == "ndarray": # image is from numpy/scipy
-        out_image = Image.fromarray(image)
+        if "float" in str(image.dtype) and image.min() > 0 and image.max() < 1:
+            image *= 255
```
Comment (madisonmay) [inline indicoio/utils/image.py]: `>=`, `<=`

**Comment by madisonmay** [comment]
LGTM.  Merging to development and pushing this up ASAP.

**Pull request merged**